app/views/layouts/application.html.erb

```diff
@@ -11,11 +11,10 @@
   </head>
 
   <body>
-    <%= render 'welcome/header' %>
+    <%=render 'welcome/header' %>
     <% if notice || alert %>
-      <p class="notice"><%= notice %></p>
-      <p class="alert"><%= alert %></p>
-  <% end %>
-  <%= yield %>
+      <%= render 'welcome/alert' %>
```

Comment:
You can save these partials in `layouts` folder. `welcome` folder is dedicated to only welcome page content.